dimse/dimse.go

```diff
@@ -284,58 +284,80 @@ type CommandAssembler struct {
 	commandBytes   []byte
 	command        Message
 	dataBytes      []byte
+	stream         chan []byte
```

Comment:
I'm thinking about turning `CommandAssembler` into an interface with two implementations: current and new "stream" implementation. We could

```go
type CommandAssembler interface {
  io.Writer()
}
```

Then the consumers doesn't need to know if it is in streaming mode or not, they can just calls `a.Write(item.Value)`. 

It would be nice if we have something like this, but I understand it might big refactorings, so feel free to ignore it for now.

Reply:
we don't use the old implementation anymore, so I think this is unnecessary